function-support.html

```diff
@@ -599,8 +598,16 @@ <h2 id="environments">Environments</h2>
    c &amp; d
 \end{Bmatrix}</code></pre>
             </td>
-			<td></td>
-			<td></td>
+            <td>\(\def\arraystretch{1.5}\begin{array}{c|c:c} a & b & c \\ \hline d & e & f \\ \hdashline g & h & i \end{array}\)</td>
+            <td>
+                <pre><code>\def\arraystretch{1.5}
+\begin{array}{c|c:c}
+   a &amp; b &amp; c \\ \hline
+   d &amp; e &amp; f \\
+   \hdashline
+   g &amp; h &amp; i
+\end{array}</code></pre>
+            </td>

```

Comment:
Great example!

Technically, the various other `&`s above ought to be `&amp;` in HTML. But it's been working fine, so I could go either way.